index.js

```diff
@@ -41,7 +41,8 @@ var AudioRecorder = {
       MeteringEnabled: false,
       MeasurementMode: false,
       AudioEncodingBitRate: 32000,
-      IncludeBase64: false
+      IncludeBase64: false,
+      ProgressUpdateInterval: 1000,
```

Comment:
FYI, for iOS to work you also need to update line 60 below to add `recordingOptions.ProgressUpdateInterval` to the `prepareRecordingAtPath` call.